index.html

```diff
@@ -221,6 +221,22 @@ <h3 class="head3" id="social-2">LET’S CONNECT</h3>
         </div>
       </section>
     </main>
+    <footer class="footer" id="contacts">
+      <div class="content-placeholder" id="contact-placeholder">
+        <h2 class="head-1" id="contact-me">Contact Me</h2>
+        <p id="contact-msg">
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>    
+        <form class="form" action="https://formspree.io/f/xbjwpzkw" method="post">
+          <input class="form-input input-line" type="text" placeholder="Enter your name" name="name" maxlength="30" required/>
+          <input class="form-input input-line" type="email" id="email" name="email" placeholder="yourid@gmail.com" required/>
+          <textarea class="form-input" rows="10" maxlength="500" placeholder="Write your message here" name="message" required ></textarea>
+          <input class="btn" type="submit" value="Get It Touch" />
```

Comment:
- [ ] Kindly align left the button as in the Figma design for mobile version. On the desktop version you will have it at the center, but is in your next milestone.
### Design
![contact-form-button-left-figma](https://user-images.githubusercontent.com/44057863/166562177-6a9697b5-df9d-4525-8011-67f4a3a97e1f.png)
### Impl
![contact-form-button-centered-impl](https://user-images.githubusercontent.com/44057863/166562200-53230156-b565-46b9-a42b-6250308c5020.png)

- [ ] Kindly use the background color during `active` state (while the button is pressed) is `#2230D2`.
```
btn:active {
}
```
![pressed-button-bg](https://user-images.githubusercontent.com/44057863/166562446-17183fae-253a-412d-af60-afbb045b28df.png)


- [ Optional ] It is good if you can pointer cursor when you hover on the button for a better user experience by indicating it is clickable. `cursor: pointer;`